### PR TITLE
Move tests into a tests folder

### PR DESCRIPTION
**Pull request by gaetbout**
Moving tests from the contracts folder to a sub folder tests

### CODE REVIEW

**Review by Leonard-Pat — APPROVED** [review]

File: contracts/tests.cairo [hunk 1]
```diff
@@ -0,0 +1,3 @@
+mod test_asserts;
```
Comment (delaaxe) [inline contracts/tests.cairo:1]: is this file necessary?

**Review by delaaxe — APPROVED** [review]

File: contracts/tests.cairo (hunk 1 above)
Comment (gaetbout) [reply contracts/tests.cairo:1]: At the moment yes, because their pipeline expects to find a file called tests.cairo that will then itself call the test files.
Quite annoying, I agree.

**Review by juniset — APPROVED** [review]
